index.js

```diff
@@ -17,7 +17,7 @@ module.exports = postcss.plugin("postcss-color-function", function() {
       }
 
       decl.value = helpers.try(function transformColorValue() {
```

Comment:
There is a test for throwing a readable error that passes, even though I removed the try block. I couldn't quite figure out why it passes. I assumed it was because colorFn.convert() is the one that throws the error.